test/listener_test.js

```diff
@@ -1,5 +1,5 @@
-suite('TaskListener', () => {
-  test('listens', async () => {
+suite('TaskListener', function() {
+  test('listens', async function() {
```

Comment:
This change is key to `this.skip()` working -- arrow functions don't take a context (`this`), while `function` does.  In most cases, we use arrow functions these days, but for Mocha tests and suites, it's best to use `function`.

Reply:
Yes, it's probably I forgot to save after changing arrow to function - and got the error.